nimbus.fml.yaml

```diff
@@ -33,6 +34,12 @@ features:
             "contile-top-sites": false,
           }
         }
+      - channel: developer
+        value: {
+          "sections-enabled": {
+            "pocket-sponsored-stories": true,
+          }
+        }
```

Comment:
From https://experimenter.info/fml-spec/ I understand this patches the default configuration and so there's no need to provide values for other sections for which I'm not interested to change their behavior.

Reply:
👍🏽